roles/operator_build/defaults/main.yml

```diff
@@ -16,8 +16,8 @@
 
 cifmw_operator_build_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}"
 cifmw_operator_build_dryrun: false
-cifmw_operator_build_golang_ct: "docker.io/library/golang:1.19"
-cifmw_operator_build_golang_alt_ct: "quay.rdoproject.org/openstack-k8s-operators/golang:1.19"
+cifmw_operator_build_golang_ct: "docker.io/library/golang:1.20"
+cifmw_operator_build_golang_alt_ct: "quay.rdoproject.org/openstack-k8s-operators/golang:1.20"
```

Comment:
Yeah, let's stick with Go 1.20 because we can't upgrade to 1.21 until there's a FIPs compliant UBI9 build available to us.